### PR TITLE
Updated CC version + minor refactors

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#10922

*Description of changes:*
* Updated CC version
* Minor refactors to resolve Unit test breakages

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -151,7 +151,8 @@ import * as Sentry from '@sentry/browser'
 import { getFeatureFlag, updateLdUser } from '@/utils'
 
 // Components
-import { PaySystemAlert, SbcFeeSummary, SbcFooter, SbcHeader } from '@/components'
+import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
+import { PaySystemAlert, SbcFeeSummary, SbcFooter } from '@/components'
```
Comment (severinbeauvais) [inline src/App.vue:155]: Isn't this equivalent to the previous code?

Comment (severinbeauvais) [reply src/App.vue:155]: NVM - I see what you've done.

Why did you do this?

**Review by severinbeauvais — APPROVED** [review]
Must be \*\*magic\*\* because I don't see how this fixed the test breakage...?

File: src/App.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/App.vue:155]: Something has changed on the components side, that impacted how we import/exported out of our common index. 
I imagine it's more to do with how we create a reference for it locally, ie `export ( default as ... )`
I don't know how/why at this time. Will look into this some more though in the morning as per the explanation. 
Perhaps there is a better way to do this!

**Comment by cameron-freshworks** [comment]
> Must be **magic** because I don't see how this fixed the test breakage...?

Magic indeed! 
Something wasn't resolving in the test with regards to the sbc-Header obviously. 
It's not the header component itself but rather it's sub components. 
Perhaps my solution was too heavy handed and i can just import directly for Unit tests, since thats the time we were getting errors. 
Will look into this more! 👍

File: src/App.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/App.vue:155]: Not quite, the import/export reference we created in our index file would no longer work.
I've yet to discover the reason why a direct import resolves but looking closer at this.

**Pull request merged**